--- Valentina.pri.orig	2015-05-14 18:47:29.704026058 +0200
+++ Valentina.pri	2015-05-14 18:48:03.100790593 +0200
@@ -70,24 +70,6 @@
 }
 
 defineReplace(enable_ccache){
-    # ccache support only Unix systems.
-    unix:{
-        # This need for turn on ccache.
-        *-g++{
-            QMAKE_CC = ccache gcc
-            export(QMAKE_CC) # export value to global variable.
-
-            QMAKE_CXX = ccache g++
-            export(QMAKE_CXX) # export value to global variable.
-        }
-        clang*{
-            QMAKE_CC = ccache clang
-            export(QMAKE_CC) # export value to global variable.
-
-            QMAKE_CXX = ccache clang++
-            export(QMAKE_CXX) # export value to global variable.
-        }
-    }
     return(true)
 }
 
