diff -urN src.orig/app/app.pro src/app/app.pro
--- src.orig/app/app.pro	2015-05-04 10:52:56.000000000 +0200
+++ src/app/app.pro	2015-05-15 00:29:11.488665402 +0200
@@ -572,25 +572,6 @@
 else:unix|win32-g++: PRE_TARGETDEPS += $$OUT_PWD/../libs/vlayout/$${DESTDIR}/libvlayout.a
 
 
-# Strip after you link all libaries.
-CONFIG(release, debug|release){
-    win32:!win32-msvc*{
-        # Strip debug symbols.
-        QMAKE_POST_LINK += objcopy --only-keep-debug bin/${TARGET} bin/${TARGET}.dbg &&
-        QMAKE_POST_LINK += objcopy --strip-debug bin/${TARGET} &&
-        QMAKE_POST_LINK += objcopy --add-gnu-debuglink="bin/${TARGET}.dbg" bin/${TARGET}
-    }
-
-    unix:!macx{
-        # Strip debug symbols.
-        QMAKE_POST_LINK += objcopy --only-keep-debug ${TARGET} ${TARGET}.dbg &&
-        QMAKE_POST_LINK += objcopy --strip-debug ${TARGET} &&
-        QMAKE_POST_LINK += objcopy --add-gnu-debuglink="${TARGET}.dbg" ${TARGET}
-    }
-}
-
-
-
 macx{
    # run macdeployqt to include all qt libraries in packet
    QMAKE_POST_LINK += $$[QT_INSTALL_BINS]/macdeployqt $${OUT_PWD}/$${DESTDIR}/$${TARGET}.app
diff -urN src.orig/libs/qmuparser/qmuparser.pro src/libs/qmuparser/qmuparser.pro
--- src.orig/libs/qmuparser/qmuparser.pro	2015-05-04 10:52:56.000000000 +0200
+++ src/libs/qmuparser/qmuparser.pro	2015-05-15 00:29:32.708737993 +0200
@@ -105,10 +105,5 @@
         QMAKE_CXXFLAGS_RELEASE += -g -gdwarf-3
         QMAKE_CFLAGS_RELEASE += -g -gdwarf-3
         QMAKE_LFLAGS_RELEASE =
-
-        # Strip debug symbols.
-        QMAKE_POST_LINK += objcopy --only-keep-debug bin/${TARGET} bin/${TARGET}.dbg &&
-        QMAKE_POST_LINK += objcopy --strip-debug bin/${TARGET} &&
-        QMAKE_POST_LINK += objcopy --add-gnu-debuglink="bin/${TARGET}.dbg" bin/${TARGET}
     }
 }
diff -urN src.orig/libs/vpropertyexplorer/vpropertyexplorer.pro src/libs/vpropertyexplorer/vpropertyexplorer.pro
--- src.orig/libs/vpropertyexplorer/vpropertyexplorer.pro	2015-05-04 10:52:56.000000000 +0200
+++ src/libs/vpropertyexplorer/vpropertyexplorer.pro	2015-05-15 00:29:44.295444295 +0200
@@ -114,10 +114,5 @@
         QMAKE_CXXFLAGS_RELEASE += -g -gdwarf-3
         QMAKE_CFLAGS_RELEASE += -g -gdwarf-3
         QMAKE_LFLAGS_RELEASE =
-
-        # Strip debug symbols.
-        QMAKE_POST_LINK += objcopy --only-keep-debug bin/${TARGET} bin/${TARGET}.dbg &&
-        QMAKE_POST_LINK += objcopy --strip-debug bin/${TARGET} &&
-        QMAKE_POST_LINK += objcopy --add-gnu-debuglink="bin/${TARGET}.dbg" bin/${TARGET}
     }
 }
diff -urN src.orig/test/ParserTest/ParserTest.pro src/test/ParserTest/ParserTest.pro
--- src.orig/test/ParserTest/ParserTest.pro	2015-05-04 10:52:56.000000000 +0200
+++ src/test/ParserTest/ParserTest.pro	2015-05-15 00:29:53.118807812 +0200
@@ -89,13 +89,3 @@
 
 INCLUDEPATH += $$PWD/../../libs/qmuparser
 DEPENDPATH += $$PWD/../../libs/qmuparser
-
-# Strip after you link all libaries.
-CONFIG(release, debug|release){
-    unix:!macx{
-        # Strip debug symbols.
-        QMAKE_POST_LINK += objcopy --only-keep-debug $(TARGET) $(TARGET).debug &&
-        QMAKE_POST_LINK += strip --strip-debug --strip-unneeded $(TARGET) &&
-        QMAKE_POST_LINK += objcopy --add-gnu-debuglink $(TARGET).debug $(TARGET)
-    }
-}
